src/Traveler.js

```diff
@@ -1,9 +1,28 @@
+// const moment = require('moment')
+import moment from 'moment';
+
+
 class Traveler {
-  constructor(id, name, travelerType) {
-    this.id = id;
-    this.name = name;
-    this.travelerType = travelerType;
+  constructor(traveler, trips) {
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+    this.allTrips = trips;
+    this.travelersTrips = [];
+  }
+
+  findTrips() {
+    this.travelersTrips = this.allTrips.filter(trip => trip.userID === this.id)
+  };
```

Comment:
I think if you wanted to refactor, you could separate this out into a some kind of TripRepository. That being said, the word "Repository" makes me physically ill.

Reply:
LOL